crates/ruma-federation-api/src/membership/create_invite/v2.rs

```diff
@@ -57,8 +67,17 @@ impl Request {
         room_version: RoomVersionId,
         event: Box<RawJsonValue>,
         invite_room_state: Vec<Raw<AnyStrippedStateEvent>>,
+        #[cfg(feature = "unstable-msc4125")] via: Option<Vec<OwnedServerName>>,
```

Comment:
We cannot set a function parameter behind a cargo feature, that makes the feature non-additive. I believe we should just default to an empty `Vec`, that will be mutated after construction if needed.

Reply:
See [my other comment](https://github.com/ruma/ruma/pull/1773#discussion_r1559484661) as to why this cannot be done.

Reply:
In that case, default to `None`.